packages/terra-alert/src/Alert.module.scss

```diff
@@ -29,36 +33,39 @@
     flex: 1 1 auto;
     flex-direction: row;
     line-height: 1.2;
-    margin-left: 0.6667rem;
-    margin-top: 0.7143rem;
+    margin-left: var(--terra-alert-margin-left, 0.64285rem);
```

Comment:
nit: I'd add `body` to these, `--terra-alert-body-margin-left`, in case we ever need margin on the `<div class="alert>` element.

Reply:
https://github.com/cerner/terra-core/pull/2133/commits/a72fdf778b090b7cf5ac106e1f1ecd0bd1456945#diff-3b62f1ed274440a76fc2e27af8d4700bR36